### DISCUSSION TITLE
Walls

### DISCUSSION THREAD

**Discussion by Lolopoloc** [Roadhog360/Et-Futurum-Requiem#281] [Improvements]
Could you make walls, glass and fences connect with eachother? 

Also walls without gaps between

**Comment by Roadhog360** [comment]
I have actually considered this, but the gapless walls idea is incredibly difficult to do without block states. I attempted to add this to the game at some point, but gave up.

**Comment by GregoriusT** [comment]
They could be made to connect with each other using mixins or asm, afterall they already have a check to connect with their own type, one would just need to rewrite that check to make them connect with ANY type that newer vanilla lets them connect to.

**Comment by Roadhog360** [comment]
Oh, the part that allows them to connect is the easy part. But without gapless walls, it makes it all pointless. The gaps are the hard part, but I'm not really sure what to do about those ugly things.
